docs/manual-EN/1.overview/2.quick-start/1.get-started.md

```diff
@@ -1,306 +1,452 @@
+Now you can check the tags and edge types you just created.
+
+To show the tags you just created, enter the following statement:
 
 ```ngql
--- Show tag list
 nebula> SHOW TAGS;
+```
+
+The following information is displayed:
+
+```
 ============
 | Name     |
 ============
-| building |
-------------
-| course   |
+| player   |
 ------------
-| student  |
+| team     |
 ------------
--- Show edge type list
+```
+
+To show the edge types you just created, enter the following statement:
+
+```ngql
 nebula> SHOW EDGES;
+```
+
+The following information is displayed:
+
+```
 ==========
 | Name   |
 ==========
-| choose |
+| serve |
 ----------
 | follow |
 ----------
 ```
 
-To show the attributes of a tag or an edge type:
+To show the attributes of the **player** tag, enter the following statement:
 
 ```ngql
--- Show attributes of a tag
-nebula> DESCRIBE TAG student;
+nebula> DESCRIBE TAG player;
+```
+
+The following information is displayed:
+
+```
 ===================
 | Field  | Type   |
 ===================
 | name   | string |
 -------------------
 | age    | int    |
 -------------------
-| gender | string |
--------------------
--- Show attributes of an edge type
+```
+
+To show the attributes of the **follow** edge type, enter the following statement:
+
+```ngql
 nebula> DESCRIBE EDGE follow;
+```
+
+The following information is displayed:
+
+```
 =====================
 | Field    | Type   |
 =====================
-| likeness | double |
+| degree   | int    |
 ---------------------
 ```
 
-### Insert Data
+### Inserting Data
+
+You can insert vertices and edges based on relations in the illustration figure.
+
+#### Inserting Vertices
+
+You can insert some vertices by the following statements:
+
+```ngql
+nebula> INSERT VERTEX player(name, age) values 100:("Time", 42);
+nebula> INSERT VERTEX player(name, age) values 101:("Tony", 36);
+nebula> INSERT VERTEX player(name, age) values 102:("LaMarcus", 33);
+nebula> INSERT VERTEX team(name) values 200:("Warriors");
+nebula> INSERT VERTEX team(name) values 201:("Nuggets");
+nebula> INSERT VERTEX player(name, age) values 121:("Useless", 60);
+```
+
+**Note**: 
+
+1. In the above vertices inserted, the number after the keyword `values` is the vertex ID (abbreviated for `VID`). The `VID` must be unique in the space.
+
+2. The last vertex inserted will be deleted in the [Deleting Data](#deleting-data) section.
+
+#### Inserting Edges
+
+You can insert some edges by the following statements:
+
+```ngql
+nebula> INSERT EDGE follow(degree) VALUES 100 -> 101:(95);
+nebula> INSERT EDGE follow(degree) VALUES 100 -> 102:(90);
+nebula> INSERT EDGE follow(degree) VALUES 102 -> 101:(75);
+nebula> INSERT EDGE serve(start_year, end_year) VALUES 100 -> 200:(1997, 2016);
+nebula> INSERT EDGE serve(start_year, end_year) VALUES 101 -> 201:(1999, 2018);
+
+```
+
+### Fetching Data
+
+After you insert some data in **Nebula Graph**, you can retrieve any data from your graph database.
```

Comment:
from your graph space

Reply:
For better understanding, I prefer database